### ISSUE TITLE
uPort for identity management

### ISSUE THREAD

**Issue by bumi** [67P/kredits-web#26]
Can we use uport for contributor identity management?
Do we then still need metamask - is it an alternative?
https://developer.uport.me/

**Comment by skddc** [comment]
Looks like we wouldn't need Metamask then: https://developer.uport.me/guides.html#signing-transactions

**Comment by bumi** [comment]
bumping this one, to look at uPort again. Seems quite some things happened since last year. 
though yes, probably uPort users would not use metamask.

**Comment by skddc** [comment]
I'd personally prefer to wait until the DIF releases code and protocol for their standardized identity solutions this summer.

**Comment by bumi** [comment]
do you have a link for that?

**Comment by skddc** [comment]
No, just following podcasts and birdsite and such. Apparently they're going to have something out by May or June.

https://identity.foundation/

**Comment by raucao** [comment]
RSK has a W3C-compatible DID solution now, called rlogin:

* https://github.com/rsksmart/rlogin
* https://developers.rsk.co/rif/rlogin/get-started/
* https://hackernoon.com/rlogin-the-new-authentication-libraries-for-blockchain-based-apps-h619330z

You can try it out here, and manually edit the data, too: https://identity.rifos.org/
